### PR TITLE
Fill out rest of V1 JSON API

### PR DESCRIPTION
**Pull request by igorkofman**
This PR addresses issue https://github.com/covid-projections/covid-data-model/issues/< >

### Please Check
- [ ] Will the *current* schema in [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.md) be updated with this PR?
- [ ] Are tests passing?

### CODE REVIEW

File: pyseir/deployment/webui_data_adaptor_v1.py [hunk 1]
```diff
@@ -195,8 +198,13 @@ def map_fips(self, fips):
...
             output_model = [[val for val in timestep.values()] for timestep in output_model.to_dict(orient='records')]
 
+            # TODO(igor): hack to get the right policy ids
             output_path = get_run_artifact_path(fips, RunArtifact.WEB_UI_RESULT, output_dir=self.output_dir)
-            output_path = output_path.replace('__INTERVENTION_IDX__', str(i_policy))
+            if is_county and i_policy > 1:
+                output_path = output_path.replace('__INTERVENTION_IDX__', str(i_policy+1))
+            else:
+                output_path = output_path.replace('__INTERVENTION_IDX__', str(i_policy))
+
```
Comment (goldblatt) [inline pyseir/deployment/webui_data_adaptor_v1.py:207]: cool okay we can remove this work-- sorry sorry for the coordination!

File: libs/functions/generate_api.py [hunk 2]
```diff
@@ -67,52 +69,55 @@ def _generate_api_for_projections(projection_row):
...
-        cumulativeDeaths=projection_row[rc.CURRENT_DEATHS],
-        peakDeaths=None,
-        peakDeathsDate=None,
-        endDate=_format_date(projection_row[rc.LAST_UPDATED])
-        + timedelta(days=FRAMES * DAYS_PER_FRAME),
     )
     return projections
 
+def _generate_actuals(projection_row, intervention, state):
+    intervention_str = {
```
Comment (goldblatt) [inline libs/functions/generate_api.py:76]: could we do this in the enum?

File: libs/functions/generate_api.py [hunk 3]
```diff
@@ -149,11 +152,12 @@ def generate_api_for_county_timeseries(projection_row, intervention, input_dir):
 
 
 def generate_api_for_state_projection_row(projection_row):
+    state_abbrev = US_STATE_ABBREV[projection_row[rc.STATE]]
     projections = _generate_api_for_projections(projection_row)
     state_result = CovidActNowStateSummary(
         lat=projection_row[rc.LATITUDE],
         long=projection_row[rc.LONGITUDE],
-        actuals=None,
+        actuals=_generate_actuals(projection_row, Intervention.FLATTEN, state_abbrev),
```
Comment (goldblatt) [inline libs/functions/generate_api.py:160]: should be intervention right?

File: libs/functions/generate_api.py [hunk 4]
```diff
@@ -163,11 +167,12 @@ def generate_api_for_state_projection_row(projection_row):
 
 
 def generate_api_for_county_projection_row(projection_row):
+    state_abbrev = US_STATE_ABBREV[projection_row[rc.STATE]]
     projections = _generate_api_for_projections(projection_row)
     county_result = CovidActNowCountySummary(
         lat=projection_row[rc.LATITUDE],
         long=projection_row[rc.LONGITUDE],
-        actuals=None,
+        actuals=_generate_actuals(projection_row, Intervention.FLATTEN, state_abbrev),
```
Comment (goldblatt) [inline libs/functions/generate_api.py:175]: should be for intervention to right not just flatten?

File: libs/functions/generate_api.py (hunk 3 above)
Comment (igorkofman) [reply libs/functions/generate_api.py:160]: face-palm!  thanks for catching these

File: libs/functions/generate_api.py [hunk 5]
```diff
@@ -67,52 +69,55 @@ def _generate_api_for_projections(projection_row):
...
+        Intervention.FLATTEN: 'stay_at_home',
+        Intervention.INFERRED: 'projected',
+        Intervention.SOCIAL_DISTANCING: 'social_distancing',
+    }[get_can_projection.get_intervention(intervention, state)]
+    return _Actuals(
+        population=projection_row[rc.POPULATION],
+        intervention=intervention_str,
+        cumulativeConfirmedCases=projection_row[rc.CURRENT_CONFIRMED],
+        cumulativeDeaths=projection_row[rc.CURRENT_DEATHS],
+        hospitalBeds = {
```
Comment (goldblatt) [inline libs/functions/generate_api.py:87]: dors this work? does it not have to be _ResourceUtilization

File: libs/enums.py [hunk 6]
```diff
@@ -7,14 +7,16 @@
...
     FLATTEN = 1 # on the webiste, strictDistancingNow
-    # FULL_CONTAINMENT = 2 # you are cancelled, but reusing this enum value
     SOCIAL_DISTANCING = 3 # weak distancingNow on the website
     CURRENT = 4  # look at what the state is and get the file for that
-    # We are using enum 2 for consistency with the website 
+    # We are using enum 2 for consistency with the website
     INFERRED = 2 # given the previous pattern, how do we predict going forward
 
+    def api_name(self):
+        return self.name()
```
Comment (goldblatt) [inline libs/enums.py:16]: does this need the `()`?

Comment (goldblatt) [reply libs/enums.py:16]: ya it should just be .name

File: libs/functions/generate_api.py (hunk 5 above)
Comment (igorkofman) [reply libs/functions/generate_api.py:82]: It does work!

**Pull request merged**